examples/09. Advanced Attributes.md

```diff
@@ -47,7 +47,7 @@ Retrieves the coupon with the given ID.
 Returns a list of your coupons.
 
 + Parameters
-    + limit = `10` (optional, number)
+    + limit: 10 (optional, number)
```

Comment:
But this should not be a problem because even though `optional` is default, the user can still say it. I am actually thinking of leaving this as it is. What do you say?